quantecon/util/tests/test_timing.py

```diff
@@ -50,10 +55,15 @@ def test_function_two_arg(n, a):
             loop_timer(5, test_function_one_arg, self.h, digits=10)
         test_two_arg = \
             loop_timer(5, test_function_two_arg, [self.h, 1], digits=10)
+
+        tol = 0.01
```

Comment:
Maybe better to use a relative tol, like `assert_allclose(actual, desired, rtol=rtol)`, rather than a hard-coded absolute tol?